tests/unittest/others/test_disagg_readiness_after_startup.py

```
@@ -0,0 +1,358 @@
+    def num_prepared_servers(self):
+        return len(self._servers)
+
+    def monitoring_is_stale(self, max_age_secs):
+        return self._stale
+
+
+def _coordinator(
+    ctx_servers,
+    gen_servers,
+    cluster_manager=None,
+    metadata_server=object(),
```

Ruff (0.16.2)

[warning] Do not perform function call `object` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)